src/features/claims/config/claimableTokens.ts

```diff
@@ -11,6 +11,7 @@ type TestnetTokensNetworkMapping = {
   [index: number]: TestnetClaimableToken[];
 };
 
+// FIXME: replace Goerli with Sepoli (11155111) addresses
```

Comment:
Let's add the following for Sepolia.
"0x20aaebad8c7c6ffb6fdaa5a622c399561562beea", // USDT
"0xf450ef4f268eaf2d3d8f9ed0354852e255a5eaef", // USDC

Reply:
I've added these in. Please double check the setup